site2/docs/admin-api-topics.md

```diff
@@ -1686,45 +1740,16 @@ You can check the current statistics of a given topic. The following is an examp
 }
 ```
 
-You can check the current statistics of a given topic and its connected producers and consumers in the following ways.
-
-````mdx-code-block
-<Tabs groupId="api-choice"
-  defaultValue="pulsar-admin"
-  values={[{"label":"pulsar-admin","value":"pulsar-admin"},{"label":"REST API","value":"REST API"},{"label":"Java","value":"Java"}]}>
-<TabItem value="pulsar-admin">
-
-```shell
-pulsar-admin topics stats \
-persistent://test-tenant/namespace/topic \
---get-precise-backlog
-```
-
-</TabItem>
-<TabItem value="REST API">
-
-{@inject: endpoint|GET|/admin/v2/:schema/:tenant/:namespace/:topic/stats|operation/getStats?version=@pulsar:version_number@}
-
-</TabItem>
-<TabItem value="Java">
-
-```java
-admin.topics().getStats(topic, false /* is precise backlog */);
-```
-
-</TabItem>
-
-</Tabs>
-````
-
 ## Manage partitioned topics
 You can use Pulsar [admin API](admin-api-overview.md) to create, update, delete and check the status of partitioned topics.
 
 ### Create
 
-Partitioned topics must be explicitly created. When creating a new partitioned topic, you need to provide a name and the number of partitions for the topic.
+When creating a new partitioned topic, you need to provide a name and the number of partitions for the topic.
 
-By default, 60 seconds after creation, topics are considered inactive and deleted automatically to avoid generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to a specific value.
+:::note
+By default, if there are no messages 60 seconds after creation, topics are considered inactive and deleted automatically to avoid generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to a specific value.
+:::
```

Comment:
```suggestion
:::note

By default, if there are no messages 60 seconds after creation, topics are considered inactive and deleted automatically to avoid generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to a specific value.

:::
```

You need empty lines here, see also:

* https://pulsar.apache.org/contribute/document-syntax#admonitions
* https://pulsar.apache.org/contribute/document-syntax#indentation-and-space

Also, please [preview your changes](https://pulsar.apache.org/contribute/document-preview). Such false rendering should be found at the first glance if you preview them.

Reply:
By the way, why do we need empty lines? I think empty lines may cause a rendering bug in a list, see https://pulsar.apache.org/docs/next/schema-overview#producer-side

<img width="896" alt="image" src="https://user-images.githubusercontent.com/37220920/202878645-7f0f1e61-213c-4082-bb44-d60e3aba36eb.png">

I have tested,  if remove empty lines can fix this bug.

Reply:
OK I found that the root cause is not about empty lines, it's caused by a redundant space : )

Reply:
:)

You can read the syntax guide and jump to MDX/Docusaurus references:

* https://pulsar.apache.org/contribute/document-syntax
* https://docusaurus.io/docs/markdown-features